File: API/Models/JobRequest.cs
namespace All4SA.Models
{
    public class JobRequest
    {
        public int JobRequestID { get; set; }
        public string JobRequestDescription { get; set; }
        public int UserID { get; set; }
        public int ImageReferenceID { get; set; }
        public int JobTypeID { get; set; }
        public decimal EstimatedCost { get; set; }
        public bool Status { get; set; }
    }
}

File: AngularUI/src/app/interfaces/jobRequest.ts
export interface jobRequest {
    jobRequestID: number,
    jobRequestDescription: string,
    userID: number,
    imageReferenceID: number,
    jobTypeID: number,
    estimatedCost: number,
    status: boolean
}

// {
//     "jobRequestID": 0,
//     "jobRequestDescription": "string",
//     "userID": 0,
//     "imageReferenceID": 0,
//     "jobTypeID": 0,
//     "estimatedCost": 0,
//     "status": true
//   }